easybuild/tools/job/gc3pie.py

```diff
@@ -0,0 +1,205 @@
+##
+# Copyright 2015-2015 Ghent University
+# Copyright 2015 S3IT, University of Zurich
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# the Hercules foundation (http://www.herculesstichting.be/in_English)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+Interface for submitting jobs via GC3Pie.
+
+@author: Riccardo Murri (University of Zurich)
+"""
+
+
+import os
+import time
+
+try:
+    import gc3libs
+    from gc3libs import Application, Run, create_engine
+    from gc3libs.core import Engine
+    from gc3libs.quantity import hours as hr
+    from gc3libs.workflow import DependentTaskCollection
+    HAVE_GC3PIE = True
+except ImportError:
+    HAVE_GC3PIE = False
+
+from easybuild.tools.build_log import print_msg
+from easybuild.tools.job import JobServer
+
+from vsc.utils import fancylogger
+
+# inject EasyBuild logger into GC3Pie
+gc3libs.log = fancylogger.getLogger('gc3pie', fname=False)
+# make handling of log.error compatible with stdlib logging
+gc3libs.log.raiseError = False
```

Comment:
We've deprecated the `log.error` method throwing exceptions, but it's still there (and will be until EB v3.0).

We'll still need a workaround, I'll figure something out in EB for that.

Maybe this will suffice.